.gitignore

```diff
@@ -27,6 +27,8 @@ hs_err_pid*
 # Gradle
 .gradle
 build/
+**/bin/main/
+**/bin/test/
```

Comment:
What is this for?

Reply:
A lot of files are put into the bin folder in each subproject whenever I run gradle build (I believe that is a VSCode thing as shown here: https://github.com/microsoft/vscode-gradle/blob/develop/.gitignore)